tests/test_butler.py

```diff
@@ -82,13 +82,13 @@ def assertExposure(self, exp, ccd, checkFilter=True):
         print("width: ", exp.getWidth())
         print("height: ", exp.getHeight())
         print("detector name: ", exp.getDetector().getName())
-        print("filter name: ", exp.getFilter().getFilterProperty().getName())
+        print("filter name: ", exp.getFilter().getName())
```

Comment:
As in lsst/pipe_tasks#428, I'm pretty sure this is an idiom for `getCanonicalName()` that predates its introduction.